src/crypto/index.js

```diff
@@ -889,37 +922,8 @@ Crypto.prototype.onCryptoEvent = async function(event) {
 Crypto.prototype.onSyncCompleted = async function(syncData) {
     const nextSyncToken = syncData.nextSyncToken;
 
-    if (!syncData.oldSyncToken) {
-        console.log("Completed initial sync");
-
-        // if we have a deviceSyncToken, we can tell the deviceList to
-        // invalidate devices which have changed since then.
-        const oldSyncToken = this._sessionStore.getEndToEndDeviceSyncToken();
-        if (oldSyncToken !== null) {
-            try {
-                await this._invalidateDeviceListsSince(
-                    oldSyncToken, nextSyncToken,
-                );
-            } catch (e) {
-                // if that failed, we fall back to invalidating everyone.
-                console.warn("Error fetching changed device list", e);
-                this._deviceList.invalidateAllDeviceLists();
-            }
-        } else {
-            // otherwise, we have to invalidate all devices for all users we
-            // are tracking.
-            console.log("Completed first initialsync; invalidating all " +
-                        "device list caches");
-            this._deviceList.invalidateAllDeviceLists();
```

Comment:
This part is the initialsync handling that was removed in favour of just refreshing everything, similar to what we'd do on a login (except here we have device data which is retained).